index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
you renamed the file and git is showing a green "**new**" `index.html` file, but it's not showing a "**deleted**" `rock_paper_scissors.html` file. Could it be that you forgot to commit it?

Since it's the last commit, you can just do:
```
git add rock_paper_scissors.html
git commit --amend
git push origin part_3_4 -f
```
(the -f means force, it's because you updated the last commit)
once you do this you will see how git marks it as just renamed rather than showing a green file :+1: